src/utils/TxmaEvent.ts

```diff
@@ -29,13 +32,19 @@ export interface BaseTxmaEvent {
 export interface TxmaEvent extends BaseTxmaEvent {
 	"event_name": TxmaEventName;
 	"restricted"?: VerifiedCredential["credentialSubject"];
+	"extensions"?: { evidence: VerifiedCredentialEvidence };
+	"email"?: string;
+	"document_type"?: string;
+	"issuing_country"?: string;
+	"post_office_details"?: PostOfficeInfo;
+	"document_details"?: DocumentDetails;
 }
 
 export const buildCoreEventFields = (session: ISessionItem, issuer: string, sourceIp?: string | undefined, getNow: () => number = absoluteTimeNow): BaseTxmaEvent => {
```

Comment:
I would love us to have some unit tests for this file but I'm conscious that these changes need to go in ASAP. How do you normally manage this? Do we have somewhere to add tech debt?

cc @richa-misra

Reply:
Add to the tech debt list, or make a new ticket? https://govukverify.atlassian.net/browse/F2F-919